src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -145,6 +146,7 @@ func applyClientConfig(client *client.Client, cfg *config.Config, proxyAll func(
 		Version, RevisionDate)
 	// Update client configuration
 	client.Configure(cfg.Client, proxyAll)
+	proxiedsites.Configure(cfg.ProxiedSites)
```

Comment:
Including your suggested change on the proxied sites config here @oxtoacart, as this will at least ensure we use the client-configured proxied sites initially.